.github/workflows/pr_documentation_link.yml

```
@@ -0,0 +1,31 @@
+name: Add Documentation Link to PR
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened]
+
+jobs:
+  add-doc-link:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
```

This feature would be nice! Could you update the version to `v4`? 
```suggestion
      uses: actions/checkout@v4
```

---

I couldn't get the action to work so I deleted the action